2.2/s2i/bin/assemble

```diff
@@ -43,6 +43,12 @@ if [ -f Gemfile ]; then
   bundle clean -V
 fi
 
+if ! bundle exec rackup -h &>/dev/null; then
+  echo "WARNING: Rubygem Rack is not installed in the present image."
+  echo "         You might consider to install it by adding to"
+  echo "         your Gemfile, in order to start the web server"
```

Comment:
change this (everywhere) to:
"Add rack to your Gemfile in order to start the web server."